__tests__/app.test.js

```diff
@@ -77,3 +77,21 @@ describe(`GET /api/reviews/:review_id`, () => {
       });
   });
 });
+
+describe(`GET /api/users`, () => {
+  test(`200: responds with an array of users`, () => {
+    return request(app)
+      .get(`/api/users`)
+      .expect(200)
+      .then(({ body }) => {
+        const users = body.users;
+        expect(users.length > 0).toBe(true);
+        expect(Array.isArray(users)).toBe(true);
```

Comment:
89 - we don't technically need this, because on line 90 we use forEach which only works on arrays, so we imply that it is an array there.